Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendPANWIoTDevicesToCiscoISE/SendPANWIoTDevicesToCiscoISE.yml

```diff
@@ -0,0 +1,32 @@
+args:
+- description: list of device maps.
+  isArray: true
+  name: device_maps
+  required: true
+- description: Active Cisco ISE Instance.
+  name: active_ise_instance
+  required: true
+comment: This script takes in custom attributes from PANW IoT cloud as an argument
+  and creates or updates endpoints in ISE with the  input custom attributes.
+commonfields:
+  id: SendPANWIoTDevicesToCiscoISE
+  version: -1
+dockerimage: demisto/python3:3.8.6.13358
+enabled: true
+name: SendPANWIoTDevicesToCiscoISE
+outputs:
+- contextPath: PaloAltoIoTIntegrationBase.Status
+  description: total count of devices updated or created on ISE
+runas: DBotRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- PANW IoT 3rd Party Integration
+- Cisco ISE
+timeout: 362.88ms
```

Comment:
can you explain why this is needed?

Reply:
Do you mean this script/automation? This script does the following at a high level:
- takes in Cisco ISE endpoint custom attributes via input and updates each endpoint in ISE.
I chose to do this in a script because there is a lot of logic involved and I felt it will be convenient to do in a script.
We can discuss more offline.

Reply:
I meant the timeout value is rather high. let's discuss.